charts/kserve-resources/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-name: kserve 
-version: v0.14.0
+name: kserve
+version: v0.14.1
```

Comment:
Is this intentional ?

Reply:
It was however, looking back at past commits I don't think it was nessecarry, I have removed it